collections/ansible_collections/cloudkit/service/roles/cleanup_stale_network_resources/tasks/cleanup.yaml

```diff
@@ -0,0 +1,256 @@
+---
+- name: Get all existing ClusterOrders from specified namespace
+  kubernetes.core.k8s_info:
+    api_version: cloudkit.openshift.io/v1alpha1
+    kind: ClusterOrder
+    namespace: "{{ cleanup_stale_network_resources_instance_name }}"
+  register: cluster_orders
+  failed_when: false
+
+- name: Initialize existing cluster names list
+  ansible.builtin.set_fact:
+    existing_cluster_names: []
+
+- name: Extract cluster names from ClusterOrders
+  ansible.builtin.set_fact:
+    existing_cluster_names: "{{ existing_cluster_names + ['clusterorder_' + item.metadata.name] }}"
+  loop: "{{ cluster_orders.resources }}"
+  when: cluster_orders.resources is defined
+
+- name: Display existing cluster names
+  ansible.builtin.debug:
+    msg: |
+      Found existing ClusterOrders: {{ existing_cluster_names }}
+
+- name: Initialize stale resources lists
+  ansible.builtin.set_fact:
+    stale_networks: []
+    stale_floating_ips: []
+    stale_routers: []
+    stale_subnets: []
```

Comment:
These could be set in `vars/main.yaml` instead, since we're initializing them with static values.